pages/docs/index.html

```diff
@@ -3,5 +3,144 @@
+  can install updates without a reboot, and just restart affected
+  applications afterwards to get the new version.</p>
+
+  <h3>Check for application updates?</h3>
+
+  <p>This command will let you check if updates are available: <em>flatpak remote-ls --updates</em>.
+  if you add the -d option, it will generate detailed information about the available
+  updates:
+
+<pre>
+$ flatpak remote-ls --updates -d
+Ref                                                     Origin             Commit       Installed size Download size
+app/com.valvesoftware.Steam/x86_64/stable               flathub            effe21b05e42   6.3 MB         4.1 MB
+app/org.blender.Blender/x86_64/stable                   flathub            ac90294a1cc3 225.4 MB        79.6 MB
+runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6 flathub            818a1504fad1   8.7 MB         2.7 MB
+runtime/org.gnome.Platform/x86_64/3.24                  flathub            6132154b089b 739.5 MB       281.4 MB
+runtime/org.gnome.Platform/x86_64/3.26                  flathub            be9fe7b43711 782.5 MB       296.8 MB
+runtime/org.gnome.Sdk/x86_64/3.26                       flathub            c62d5ddf669a   1.6 GB       582.9 MB
+runtime/org.kde.Platform/x86_64/5.9                     flathub            ea891f4a918f   1.1 GB       391.0 MB
+app/org.gnome.Builder/x86_64/master                     gnome-apps-nightly b14e6c0fc6c3 105.0 MB        35.4 MB
+</pre>
+
+  gnome-software will also notify you about application updates, just like it does
+  on the normal workstation.</p>
+
+  <h3>Get onto a new OS release? Do I have to reinstall?</h3>
+
+  <p>No, you don't have to reinstall. You can run: <em>rpm-ostree rebase</em>,
+  and give it the name of the new OS (fixme: how to find that name).
+  The process is very similar to a system update: the new OS is
+  downloaded and installed in the background, and you just boot into
+  it when it is ready.</p>
+
+  <p>We plan to have gnome-software offer the same functionality in
+  Fedora 28 as well.</p>
+
+  <h3>Get my vpn connections back?</h3>
+
+  <p>/etc is not part of the immutable OS image, so you can just copy
+  files into /etc/NetworkManager/system-connections (or let NetworkManager
+  store them there when you recreate your connections). Certificates
+  in /etc/pki need to be handled similarly.</p>
+
+  <h3>Install app xyz?</h3>
+
+  <p>gnome-software has excellent support for installing flatpaks. A convenient
+  place to find many popular desktop apps as flatpaks is <a href="flathub.org">flathub.org</a>.</p>
+
+  <h3>Get my browser to play youtube videos?</h3>
+
+  <p>firefox is included in the Atomic Workstation image. Getting it to
+  play videos works the same way as it does for the regular Workstation:
+  find a package with the needed codecs, and install it. The one difference
+  is that you'll use rpm-ostree install instead of dnf install.</p>
+
+  <h3>Install gcc or gdb?</h3>
+
+  <p>See 'pet container'.</p>
```

Comment:
There is no pet container entry anywhere yet (I suspect this is know, just adding it so we do not forget)